pkg/crcontrollers/operator/controller.go

```diff
@@ -455,6 +457,9 @@ func (bmc *Controller) updateNodeLabelsAndAnnotation(k8sNode *coreV1.Node, goalV
 			k8sNode.Labels[common.NodeKernelVersionLabelKey] = version
 			toUpdate = true
 		}
+		if bmc.observer != nil {
+			bmc.observer.Notify(version)
```

Comment:
Do we really need to notify each time? Does it make sense to do this only when kernel label isn't set or changed?